tests/bearlib/languages/documentation/DocumentationCommentTest.py

```diff
@@ -168,3 +168,25 @@ def test_java_default(self):
                          desc='     the concatenated string\n')]]
 
         self.assertEqual(expected, parsed_docs)
+
+
+class AssembledDocumentationTest(unittest.TestCase):
+
+    def test_python_assembly(self):
```

Comment:
um, assembling documentation is a different thing, wouldn't it be better if it was a different class.